Fix:

conftest.c:29:7: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2010-09-16 23:07:35.000000000 -0500
+++ configure	2021-02-11 17:38:57.000000000 -0600
@@ -4825,8 +4825,8 @@
   for (i = 0; i < 256; i++)
     if (XOR (islower (i), ISLOWER (i))
 	|| toupper (i) != TOUPPER (i))
-      exit(2);
-  exit (0);
+      return 2;
+  return 0;
 }
 _ACEOF
 rm -f conftest$ac_exeext
